mlflow/tracking/fluent.py

```diff
@@ -521,10 +835,50 @@ def list_run_infos(
     :param experiment_id: The experiment id which to search
     :param run_view_type: ACTIVE_ONLY, DELETED_ONLY, or ALL runs
     :param max_results: Maximum number of results desired.
-    :param order_by: List of order_by clauses.
+    :param order_by: List of order_by clauses. Currently supported values are
+           are ``metric.key``, ``parameter.key``, ``tag.key``, ``attribute.key``.
+           For example, ``order_by=["tag.release ASC", "metric.click_rate DESC"]``.
 
     :return: A list of :py:class:`mlflow.entities.RunInfo` objects that satisfy the
         search expressions.
+
+    .. code-block:: python
+        :caption: Example
+
+        from pprint import pprint
+        import mlflow
+
+        # Get run info for experiment id 0, include only active runs
+        pprint("RunInfo: {}".format(mlflow.list_run_infos(str(0), run_view_type=1)))
+
+        print("-" * 80)
+
+        # Get run info for experiment id 0, include all runs
+        # order_by types are ['metric.key', 'parameter.key', 'tag.key', 'attribute.key']
+        pprint("RunInfo: {}".format(mlflow.list_run_infos(str(0), run_view_type=3,
+                                                     order_by=["metric.click-rate DESC"])))
```

Comment:
The current code prints out empty lists if no run exists, which might confuse users. How about the example like below?

```python
import mlflow
from mlflow.entities import ViewType


# Create two runs
with mlflow.start_run() as run1:
    mlflow.log_param("p", 0)

with mlflow.start_run() as run2:
    mlflow.log_param("p", 1)

# Delete the last run
mlflow.delete_run(run2.info.run_id)


def print_run_infos(run_infos):
    for r in run_infos:
        print("- run_id: {}, lifecycle_stage: {}".format(r.run_id, r.lifecycle_stage))


print("Active runs:")
print_run_infos(mlflow.list_run_infos("0", run_view_type=ViewType.ACTIVE_ONLY))

print("Deleted runs:")
print_run_infos(mlflow.list_run_infos("0", run_view_type=ViewType.DELETED_ONLY))

print("All runs:")
print_run_infos(mlflow.list_run_infos("0", run_view_type=ViewType.ALL))

```

Reply:
I like it! Consider it done!